src/plugins/safeSnap/components/HandleOutcome.vue

```diff
@@ -16,6 +16,8 @@ import {
   useSafe
 } from '@/composables';
 
+import SafeSnapModalOptionApproval from './Modal/OptionApproval.vue';
```

Comment:
I didn't carefully review the two *Outcome.vue files, I just assume that this file is the original version (pre UMA changes) and the other file (that's all additions) is the post-uma changes that have already been reviewed. Lmk if it needs more careful review!

Reply:
Yep, it was just the naming that changed for these files. I confirmed `HandleOutcome.vue` matches the original version (pre-UMA changes) and `HandleOutcomeUma.vue` includes all our changes. John and I will do another check during our clean-up/review so I don't think any more review on these files is necessary. Thanks for reviewing!